resources/views/mship/waiting-lists/view.blade.php

```diff
@@ -36,6 +36,14 @@
                             @endif</td>
                     </tr>
                 </table>
+                @if(
+                    $list->pivot->current_status == 'Active' &&
```

Comment:
This is a great example of where my column based approach for tracking times and model methods would work very well.